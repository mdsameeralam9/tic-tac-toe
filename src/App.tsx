import TicTacToe from './pages/TicTac/TicTac'

function App() {
  return (
    <>
     <TicTacToe />
    </>
  )
}

export default App
