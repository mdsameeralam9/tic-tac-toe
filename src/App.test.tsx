import { render, screen } from "@testing-library/react";
import App from "./App";

describe("App Component", () => {
  it("renders TicTacToe component", () => {
    render(<App />);
    expect(screen.getByText(/O TURN/i)).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});