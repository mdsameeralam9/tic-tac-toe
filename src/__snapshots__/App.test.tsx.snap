// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`App Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="gameWrapper"
  >
    <h1>
      Tic Tac Toe Game
    </h1>
    <p>
      Play a classic game of Tic Tac Toe (also known as Noughts and Crosses) online for free. Challenge yourself or compete with a friend!
    </p>
    <div
      class="ttt-container"
    >
      <header
        class="ttt-header"
      >
        <h1
          class="ttt-turn"
        >
          O TURN
        </h1>
        <button
          aria-label="reset"
          class="ttt-reset"
          type="button"
        >
          ‚ü≤
        </button>
      </header>
      <main
        class="ttt-board"
      >
        <button
          aria-label="Cell 1, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 2, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 3, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 4, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 5, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 6, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 7, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 8, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
        <button
          aria-label="Cell 9, empty"
          aria-pressed="false"
          class="ttt-cell "
          type="button"
        />
      </main>
    </div>
  </div>
</DocumentFragment>
`;
